bee-message/src/payload/transaction/essence/regular.rs

```diff
@@ -102,6 +110,8 @@ pub struct RegularTransactionEssence {
     #[packable(verify_with = verify_inputs)]
     #[packable(unpack_error_with = |e| e.unwrap_packable_or_else(|p| Error::InvalidInputCount(p.into())))]
     inputs: BoxedSlicePrefix<Input, InputCount>,
+    /// BLAKE2b-256 hash of the serialized outputs referenced in inputs by their OutputId.
+    inputs_commitment: [u8; 32],
```

Comment:
I wonder if we eventually want to capture this information in a wrapper type. I wouldn't do it in this PR, but if you agree we could open an issue.

Reply:
Yeah I would also do that. And same for token tag which is still an array. I'll merge now but pls open an issue 👍